server/application/application.go

```diff
@@ -211,14 +210,14 @@ func (s *Server) List(ctx context.Context, q *application.ApplicationQuery) (*ap
 	filteredApps := apps
 	// Filter applications by name
 	if q.Name != nil {
-		filteredApps = argoutil.FilterByNameP(filteredApps, *q.Name)
+		filteredApps = argo.FilterByNameP(filteredApps, *q.Name)
```

Comment:
same here for me with `argoutil` vs `argo` (https://github.com/argoproj/argo-cd/pull/13889/files#r1214444966) I sightly prefer the narrowing of scope provided by `argoutil` as opposed to `argo` in terms of readability